Documentation/configuring-form-yaml.rst

```diff
@@ -88,3 +88,60 @@ The following YAML is stored as ``contact.yaml``:
           senderAddress: '{email}'
           senderName: '{name}'
           format: plaintext
+
+
+File Uploads
+------------
+
+The ``default`` preset comes with an ``FileUpload`` form element that allows the user of the form to upload arbitrary
+files.
+With the latest version, the ``EmailFinisher`` allows these files to be sent as attachments:
```

Comment:
Isn't this documentation only available when the feature is also? I would leave out "the latest version"